cmd/mesh/manifest.go

```diff
@@ -162,6 +162,20 @@ func genApplyManifests(setOverlay []string, inFilename string, force bool, dryRu
 	return nil
 }
 
+func isStderrOkay(stderr string) bool {
+	trimmedStdErr := strings.TrimSpace(stderr)
+	ignoreList := []string{
```

Comment:
better to put this list in var() block at the top to give it more visibility.

Reply:
Fixed.